src/Files.Backend.Item/Item/Library/LibraryItem.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Linq;
+
+namespace Files.Backend.Item
+{
+    internal class LibraryItem : FileItem, ILibraryItem
```

Comment:
Is the interface `ILibraryItem` for `LibraryItem` really needed? I also noticed the same behavior in other places

Reply:
Yes, the interface is required. LibraryItem is internal and therefore the dll only exposes ILibraryItem. There is therefore no dual use. This allows to have several implementations if necessary, for example to optimize, for the mockup, to add decorators for the log. If we expose in dll a class, we become its prisoner. This approach avoids unanticipated problems. For example, depending on whether the user activates the ShowFolderSize option or not, the same implementation is not returned. This makes it possible to have implementations, certainly more numerous, but simpler, more maintainable, more efficient. This avoids endless testing in implementations. We do the test only once in the factory.

Reply:
It also makes it easier to test the application, by sending fake data to test the interface or easily sending edge cases to test stability.